dev_envs/mac-env-setup.md

```diff
@@ -0,0 +1,260 @@
+# Setting up a Mac-based development environment #
+
+## Initial account setup (as administrator) ##
+
+For this section, log in as your administrator account.
+
+### Account permissions ###
+
+You’ll need to allow your standard user access to run `sudo` commands.
+First, you’ll want to take note of your account name by viewing the folders
+listed inside of the `/Users/` folder - in this example, the account
+name is **username**.
+
+```bash
+$ ls /Users/
+username        Shared        fedadmin
+```
+
+Then, you’ll add your **username** to the `/etc/sudoers` file using `visudo`:
+
+```sh
+$ sudo visudo
+...
+##
+# User specification
+##
+
+# root and users in group wheel can run anything on any machine as any user
+root        ALL = (ALL) ALL
+%admin      ALL = (ALL) ALL
+username    ALL = (ALL) ALL
+```
+
+Press `Esc` followed by `:wq` to save and quit `visudo`.
+
+### Install XCode command line tools ###
+
+Download XCode and Accept the License Agreement, then install the XCode
+command line tools.
+
+```bash
+sudo xcodebuild -license
+xcode-select --install
+```
+
+### Docker ###
+
+Download the latest stable [Docker for Mac](https://www.docker.com/docker-mac).
+Install it by double-clicking the downloaded `dmg` file and dragging to the
+`Applications` folder.
+
+### iTerm2 (recommended) ###
+
+To set up iTerm2 - a MacOS terminal replacement with some nifty features -
+visit the [iTerm2 download page](https://www.iterm2.com/downloads.html) to
+get the latest stable release. Install it by double-clicking the downloaded
+`dmg` file and dragging to the `Applications` folder.
+
+Once installed, run the iTerm2 app and select the menu option
```

Comment:
Not sure if it's worth changing the verbiage here, but the app file itself is called `iTerm`, even though the app refers to itself as `iTerm2`.  It's slightly confusing either way.

Reply:
Good catch! I resolved this in one of the last five commits from 37987f9